go/adbc/driver/internal/driverbase/database.go

```diff
@@ -392,7 +415,12 @@ func newTracerProvider(exporters ...sdktrace.SpanExporter) (*sdktrace.TracerProv
 		),
 	)
 	if err != nil {
-		return nil, err
+		// If unable to merge with the default resource (conflicting ShhemaURL),
+		// use just our resource
+		mergedResource = resource.NewWithAttributes(
+			semconv.SchemaURL,
+			semconv.ServiceName(driverNamespace),
+		)
```

Comment:
are we just dropping the error? Should we log the error or otherwise only ignore it conditionally based on what the error is?

Reply:
Found the correct error handling for this situation.